css-two-column-layout/styles.css

```diff
@@ -0,0 +1,130 @@
+* {
+  box-sizing: border-box;
+}
+
+.container {
+  margin: auto;
+  width: 100%;
+  height: 500px;
+}
+
+.row {
+  text-align: center;
+  font-family: Karla, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+}
+
+.col-row {
+  font-family: Karla, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.col-whole {
+  font-weight: bold;
+}
+
+.biggest-text {
+  font-size: 70px;
+  font-family: Karla, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+}
+
+.margin-top {
+  margin-top: 70px;
+}
+
+p {
+  font-family: Karla, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  color: rgb(135 135 135);
+  font-size: 20px;
+}
+
+h1,
+h3 {
+  font-family: Karla, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 25px;
+}
+
+.header-marg {
+  padding-bottom: 40px;
+}
+
+.width {
+  margin-left: 20%;
+}
+
+.half-columns {
+  display: flex;
+  margin-left: 12%;
+}
```

Comment:
Is this supposed to be a `column-half` class? If so, its styling should only be `width: 50%`.